src/lib/drivers/smbus/SMBus.hpp

```diff
@@ -92,4 +93,6 @@ class SMBus : public device::I2C
 	 */
 	uint8_t get_pec(uint8_t *buffer, uint8_t length);
 
+	perf_counter_t _interface_errors{perf_alloc(PC_COUNT, "smbus_errors")};
```

Comment:
To keep the naming consistent I've been doing it like this everywhere.

```suggestion
	perf_counter_t _interface_errors{perf_alloc(PC_COUNT, MODULE_NAME": errors")};
```

Reply:
@dagar done